braindecode/datasets/base.py

```diff
@@ -291,3 +375,27 @@ def save(self, path, overwrite=False):
         if concat_of_raws:
             json.dump({'target_name': target_name}, open(target_file_name, 'w'))
         self.description.to_json(description_file_name)
+
+
+def get_sequence_dataset(concat_ds, seq_len, step_len=1, label_transform=None):
```

Comment:
Maybe to be more consistent on naming it would be better to use `create_sequence_dataset`?

Reply:
To me it looks like this function is quite similar to `create_windows` functions? It takes `concat_ds` and returns `BaseConcatDataset`. Maybe we should move it to `preprocessing/windowers.py`?  I feel like `datasets/base.py` is not the best place for this function but as well `preprocessing/windowers.py` may not be perfect.

Reply:
definitely also feel `create_sequence_dataset` would be nicer

Reply:
+1